src/main/java/randoop/condition/RandoopConditionError.java

```diff
@@ -0,0 +1,33 @@
+package randoop.condition;
+
+/**
+ * Exception to represent errors that occur during creation or evaluation of a {@link Condition}.
```

Comment:
Are these exceptions all bugs in Randoop, all user errors, or a combination of the two?  Please document.
If it's a combination, I suggest differentiating them so that all user errors are shown to the user and halt Randoop.